ciao-cli/main.go

```diff
@@ -27,13 +27,50 @@ import (
 	"net/http"
 	"os"
 	"strconv"
-	"time"
+	"text/template"
 
 	"github.com/golang/glog"
-
-	"github.com/01org/ciao/payloads"
 )
 
+// Item serves to represent a group of related commands
+type Item struct {
```

Comment:
@albertomurillo Sorry for the slight architectural change, but I thought about this over the weekend and I think that we should replace Item with:

``` type command struct {
             subCommands map[string]Command
}
```
and implement a complete Command interface for it (Usage, ParseArgs, Run) for it.
Run() will basically be looking for the sub-command and call into the sub-command's ParseArgs() and then Run() routine for it.